services/course-material/src/components/ContentRenderer/moocfi/ChapterProgressBlock/ChapterProgress.tsx

```diff
@@ -32,18 +32,24 @@ const ChapterProgress: React.FC<ChapterProgressProps> = ({ chapterId, courseInst
         <div
           className={css`
             width: 100%;
-            margin: 0 auto;
             text-align: center;
-            padding: 2em 0;
+            margin: 7.5em auto;
           `}
         >
           {/* TODO: Verify how it looks when score_given is a floating number */}
-          <CircularProgress
+          <Progress
+            variant="circle"
             max={getUserChapterProgress.data.score_maximum}
             given={getUserChapterProgress.data.score_given}
             point={50}
             label={t("chapter-progress")}
           />
+          <Progress
+            variant={"bar"}
+            showAsPercentage={true}
+            exercisesDone={getUserChapterProgress.data.completed_exercises}
```

Comment:
Both `exercisesDone` and `completed_exercises` are confusing names since they are really used for the number of attempted exercises. I think we should name these consistently as attempted exercises everywhere because otherwise future readers of the codebase would misunderstand the meaning of these variables/fields, leading to weird bugs.

Reply:
Yea, will rename these. :+1: